### PR TITLE
STY: Apply pyupgrade suggestions

### PR DESCRIPTION
**Pull request by DimitriPapadopoulos**
## Changes proposed in this pull request

Apply [pyupgrade](https://github.com/asottile/pyupgrade) suggestions:
```
pyupgrade --py38-plus
```

## Documentation that should be reviewed

\-

### CODE REVIEW

File: fmriprep/cli/tests/test_parser.py [hunk 1]
```diff
@@ -114,9 +114,9 @@ def _mock_check_latest(*args, **kwargs):
     captured = capsys.readouterr().err
 
     msg = """\
```
Comment (effigies) [inline fmriprep/cli/tests/test_parser.py:116]:
```suggestion
    msg = f"""\
```

Comment (effigies) [inline fmriprep/cli/tests/test_parser.py:116]:
```suggestion
    msg = f"""\
```

File: fmriprep/cli/tests/test_parser.py [hunk 2]
```diff
@@ -114,9 +114,9 @@ def _mock_check_latest(*args, **kwargs):
...
-You are using fMRIPrep-%s, and a newer version of fMRIPrep is available: %s.
+You are using fMRIPrep-{}, and a newer version of fMRIPrep is available: {}.
```
Comment (effigies) [inline fmriprep/cli/tests/test_parser.py:117]:
```suggestion
You are using fMRIPrep-{current}, and a newer version of fMRIPrep is available: {latest}.
```

File: fmriprep/cli/tests/test_parser.py [hunk 3]
```diff
@@ -114,9 +114,9 @@ def _mock_check_latest(*args, **kwargs):
...
 Please check out our documentation about how and when to upgrade:
-https://fmriprep.readthedocs.io/en/latest/faq.html#upgrading""" % (
+https://fmriprep.readthedocs.io/en/latest/faq.html#upgrading""".format(
         current,
         latest,
     )
```
Comment (effigies) [inline fmriprep/cli/tests/test_parser.py:122]:
```suggestion
https://fmriprep.readthedocs.io/en/latest/faq.html#upgrading"""
```

File: fmriprep/workflows/base.py [hunk 4]
```diff
@@ -200,8 +200,8 @@ def init_single_subject_wf(subject_id: str):
 
     if not anat_derivatives and not subject_data['t1w']:
         raise Exception(
-            "No T1w images found for participant {}. "
-            "All workflows require T1w images.".format(subject_id)
+            f"No T1w images found for participant {subject_id}. "
+            f"All workflows require T1w images."
```
Comment (effigies) [inline fmriprep/workflows/base.py:204]: I suspect this now fits on one line...

```suggestion
            f"No T1w images found for participant {subject_id}. All workflows require T1w images."
```

File: wrapper/src/fmriprep_docker/__main__.py
Comment (effigies) [inline wrapper/src/fmriprep_docker/__main__.py:1]: The wrapper is intended to be installable on the most reluctant-to-upgrade systems. We actually still write 2.7-compatible Python here.

Comment (DimitriPapadopoulos) [reply wrapper/src/fmriprep_docker/__main__.py:1]: All changes to `wrapper/src/fmriprep_docker/__main__.py` reverted.

**Review by effigies — APPROVED** [review]

**Pull request merged**